### ISSUE TITLE
Mods List

### ISSUE THREAD

**Issue by Xariyl** [Xariyl/FishyExperiment812390#1]
Open Issue for discussing the mods in the pack. Evaluations of current mods, new mod requests, version-update notices, and such.

This is not for reporting bugs! Please create a new Issue for any bugs or crashes found.

**Comment by Xariyl** [comment]
FYI: Fastcraft can also be a server mod, it has some functions that enable better synchronization between client and server.

**Comment by odenthefish** [comment]
Really?  I'll add it in the server package! The more optimization the better! Exclamation mark!

**Comment by Xariyl** [comment]
heh, you replied before I finished :) 

Fastcraft, Waila, and Wawla have some server optimization code, so yes, definitely keep them in your server-package.

**Comment by odenthefish** [comment]
Updated
botania
forestry for minecraft 
malisisdoors/core
pneumatic craft
buildcraft 
schematica

IChun util says it needs updating but it works well for now
What are we looking at might say it needs updating but it is wrong 
Project red has an update.... but the current version seems pretty stable and i need to test before the update 
Logistics pipes says it has an update but it crashes when updated (what buildcraft version could it even need???) 
i will get to the configs today

**Comment by odenthefish** [comment]
So what do you think of the mods so far? have any suggestions? i am preparing to publish the modpack (so far it has been in beta because we have been testing for crashes and because i've been lazy :/ )
i believe that all are no contact permission 
but so far but i still haven't found the license for flan's mod or the content packs 
they are all made by flan that i have so probably it is the same license as the mod its self

**Comment by Xariyl** [comment]
Yep, looks like you just need to put a link in the description of your modpack for the license.

The mods seem very stable right now, I haven't come across any conflicts or crashes. I agree and think it's ready to publish.

My one suggestion: As it is kind of a combat pack, lots of cool toys added by Flans and such, what do you think about adding a dungeon-type mod like [RoguelikeDungeons](http://www.minecraftforum.net/forums/mapping-and-modding/minecraft-mods/1290238-roguelike-dungeons-v1-3-6) or [Battletowers](http://atomicstryker.net/battletowers.php)? I've used both in the past and they're fun, though I do lean more towards Battletowers. 

Obviously this is more of a personal preference of mine, neither is needed, the pack is stable and the current mods seem to work well together.

**Comment by Xariyl** [comment]
Oops... scratch that. The latest Thermal Expansion in your dropbox pack is conflicting with Simply Jetpacks, and it crashed the server. I'm reverting back to ThermalExpansion-[1.7.10]4.0.1-182 and ThermalFoundation-[1.7.10]1.0.0-81 for now, until SJ updates.

**Comment by odenthefish** [comment]
Odd it works fine for me in my personal modpack
I'll do some testing...

**Comment by Xariyl** [comment]
Yeah, it worked on Single-player. Crashed the server though. Go figure.

**Comment by odenthefish** [comment]
Whaaa? This is why we always test on the local  server first fish brain!

**Comment by odenthefish** [comment]
Ok so i tested with even an outdated version of simply jetpacks and it still crashed the server 
oh well it wasn't that big of an update it just added a few new toys (pretty well useless for the most part)  
i will be looking at battletowers and rogelikedungons

**Comment by odenthefish** [comment]
Oi I can't go 2 days without a mod updating! 
i will have an updated buildcraft later 
i have a little over half of the permissions for the modpack so expect it to be done tonight or tomorrow 
:)

**Comment by Xariyl** [comment]
On the plus side, the last Thermal Expansion fixed the issue with Simply Jetpacks, no more server crashing :)

For pack-updates, maybe hold off on updates for a week or two? Unless there's a major bug fix, it can wait a bit.

Once the pack is finalized, updates can be spread out over maybe a month or so, to keep up with new content.

My advice for whatever it's worth.

**Comment by odenthefish** [comment]
Fine then! Make since! 
that is a pretty good idea, unless i get a chance at ultra fast internet it would be a good idea to just give it a week 
also it'd give enough time for fixes like the SJ fix 
i think i will upload a new update tomorrow and then go with the week by week update schedule

**Comment by odenthefish** [comment]
Confirmed in next update: IGW mod is back (it has support for modular turrets now and PC has new stuff that I don't entirely know how to use), traveler's gear, damage indicator mod and aroma1987's mining world.
And I have been researching Ancient warfare for the modpack, while cool its not very finished and i don't think it quite likes malisis doors... 
Maybe later, once it has vehicles again

**Comment by odenthefish** [comment]
Tested on local server and everything checked out fine so update coming later tonight.

On another note I have been looking at Archimedes ships because I just saw possibly the best ship suggestion on the minecraft forums and wanted to build a adventure ship and sail around,

but the mod coming to the pack is far down the line if ever, it isn't advanced enough and if you are not seated then you fall out... not optimal.

I've looked at the meta world's mod and, it's not quite compatible....

**Comment by Xariyl** [comment]
Archimedes Ships is one of my favorites :)  If you're going to include it, I'd recommend the version by [bk1325](http://minecraft.curseforge.com/mc-mods/228029-archimedes-ships-plus-flying-houses-flying-houses). It's more updated and includes some bug fixes. The original author, Balkondeur, hasn't updated in over a year.

**Comment by odenthefish** [comment]
Know of any versions/shipmods that you can freely walk around on? that would be so cool if you do.
I was reading this very popular suggestion on the MCF and realized how much I wished it was implemented: http://www.minecraftforum.net/forums/minecraft-discussion/suggestions/50810-boats-evolved-bigger-modular-boats 
I want a boat/ship mod that allows free movement while the ship is moving, I'm sure there would be some issues with SMP and lag but just something that can drift while the player is walking around.
I really want a way to explore massive oceans while being able to do simple things like organizing chests and admiring the horizon.
Well now I've gone off topic with my nomadic dreams.

**Comment by Xariyl** [comment]
Unfortunately it has to do with the way MC codes movement of entities. The ship-mods basically turn the ship into an entity tied to the driver. Any passengers would fall off. Along the lines of trying to walk on a vanilla boat or minecart, without being inside of it.

**Comment by odenthefish** [comment]
I suspected as much :/

**Comment by Xariyl** [comment]
BiblioCraft recently put out a bugfix patch, and RailCraft released a new version with bug fixes and a few new features. Might I suggest these for inclusion with modpack version 1.0.0a? 

:)

**Comment by odenthefish** [comment]
Yeah as soon as I updated it there was an update  :P 
The next update won't be so late.

**Comment by odenthefish** [comment]
The update WILL be this week, I'm not putting it off like I did the last update....

**Comment by odenthefish** [comment]
Thermal smeltery will be added in the next update, the update will be before the pack goes live on FTB.

**Comment by odenthefish** [comment]
Modular powersuits has updated! Do you think that MPS should be added? 
I played with it for a bit and it seems _stable_ I just messed with it for about 10 minutes but I'd like to see what you think about it being added.
http://www.curse.com/mc-mods/minecraft/235442-modular-powersuits
I also have an open PR if it doesn't display... I'm still not quite sure how it works... :P

**Comment by Xariyl** [comment]
Looks interesting. I've not used it before. I'm okay with adding it.

I get an email whenever there's a PR, and I can review the changes. I've accepted the one you just did, and once accepted it saves the changes to the master file list.

**Comment by odenthefish** [comment]
Ah I'm about to do another quick one to add MPS, I've been waiting for it for soo long :)

**Comment by odenthefish** [comment]
EDIT: after reading a bit more the creator says 
"Hello,

Please direct people to Curseforge from now on  :P Just about everything is working now for 1.7.10, so a Release version is imminent."
I'm not going to add it until i get specific permission from the Dev, some of them don't quite like unfinished versions being modpacked

**Comment by odenthefish** [comment]
Added MPS! I got an OK from the creator! :) 
Update should be later tonight.

**Comment by Xariyl** [comment]
Yay!

**Comment by odenthefish** [comment]
It's still not uploaded.. my interweb!

**Comment by odenthefish** [comment]
What do you think about this? http://www.minecraftforum.net/forums/mapping-and-modding/minecraft-mods/1291067-atg-alternate-terrain-generation
It wouldn't replace normal BoP world gen for normal play but it'd make a really good world gen for adventure servers and stuff like that.
https://www.dropbox.com/sh/czud1s5vkcyczl1/AADbR0xp_A0US_k5J2TKv01_a?dl=0
I would like to know what you think of adding it, you could still play normal BoP because its a world gen type in the more options when choosing to make a world.

**Comment by Xariyl** [comment]
Wow, this actually looks really good. I definitely vote yes.

**Comment by odenthefish** [comment]
Ok I think it'll be in the next update, although it requires a bunch of biome ID config stuff I believe.
I wouldn't recommend resetting the server for it because I think it would make a normal style survival nearly impossible. (steep cliffs, rarely flat land, and MASSIVE terrain load lag) 

But I'd say it would make a great world type for exploration and such for people with a very good  internet connection (for servers) or a good CPU (for local play)

Over all its a really cool idea that looks well implemented from what little I have seen.
